src/utils/composeStores.js

```diff
@@ -1,11 +1,18 @@
-import mapValues from '../utils/mapValues';
-import pick from '../utils/pick';
+import mapValues from './mapValues';
+import pick from './pick';
+
+// UNDEF = undefined;
+const UNDEF = void 0;
```

Comment:
This is a pretty common pattern. I will make the change. How should I fix the other UNDEF use in the test?